.agents/skills/webawesome/references/components/skeleton.md

```
@@ -0,0 +1,278 @@
+
+To import this component for React 18 or below, use the following code:
+
+```js
+import WaSkeleton from '@awesome.me/webawesome/dist/react/skeleton/index.js';
+```
+
+### Attributes & Properties
+
+| Name | Description | Reflects |
+| --- | --- | --- |
+| \`effect\` effect | \`'pulse' \\| 'sheen' \\| 'none'\` Determines which effect the skeleton will use. Type Default 'none' | |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)